Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -147,6 +159,13 @@
         volume: 12
   - type: StaticPrice
     price: 2000
+  - type: CartridgeAmmo #Sunrise-Start
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)